scipy/fft/_helper.py

```diff
@@ -0,0 +1,52 @@
+import scipy.fftpack as _fftpack
+
+def next_fast_len(target, dtype=None):
+    """Find the next fast size of input data to ``fft``, for zero-padding, etc.
+
+    SciPy's FFT algorithms gain their speed by a recursive divide and conquer
+    strategy. This relies on efficient functions for small prime factors of the
+    input length. Thus, the transforms are fastest when using composites of the
+    prime factors handled by the fft implementation. If there are efficient
+    functions for all radices <= `n` then the result is the smallest number `x`
+    >= ``target`` with only prime factors < `n`. (These are also known as
+    `n`-smooth numbers, regular numbers, or Hamming numbers.)
+
+    Parameters
+    ----------
+    target : int
+        Length to start searching from.  Must be a positive integer.
+
+    Returns
+    -------
+    out : int
+        The smallest fast length greater than or equal to ``target``.
+
+    Notes
+    -----
+    The result of this function may change if new prime factors are added.
+
+    Examples
+    --------
+    On a particular machine, an FFT of prime length takes 17 ms:
+
+    >>> from scipy import fft
+    >>> min_len = 93059  # prime length is worst case for speed
+    >>> a = np.random.randn(min_len)
+    >>> b = fft.fft(a)
+
+    Zero-padding to the next regular length reduces computation time to
+    1.3 ms, a speedup of 33 times:
```

Comment:
That seems to be a typo, should be "13 times"

Reply:
Thanks, fixed